src/handlers/testing/exec/util/ci.ts

```diff
@@ -59,6 +59,8 @@ const zRepositorySchema = z.object({
 const zAutoblocksOverridesSchema = z.object({
   // Map of prompt external ID to a prompt revision ID
   promptRevisions: z.record(z.string(), z.string()),
+  // Map of config external ID to a config revision ID
+  configRevisions: z.record(z.string(), z.string()),
```

Comment:
Is this always required or can it be optional?

Reply:
ya these should both be optional b/c you might use just one or both